src/ClientData/CaptureData.cpp

```diff
@@ -144,6 +144,12 @@ void CaptureData::ComputeVirtualAddressOfInstrumentedFunctionsIfNecessary(
         module_data->ConvertFromOffsetInFileToVirtualAddress(instrumented_function.file_offset());
     instrumented_function.set_function_virtual_address(virtual_address);
     ++updated_function_count;
+
+    ORBIT_CHECK(scope_id_provider_);
+    auto scope_id = scope_id_provider_->FunctionIdToScopeId(function_id);
+    if (scope_id.has_value()) {
+      scope_id_provider_->UpdateFunctionInfoAddress(scope_id.value(), virtual_address);
+    }
```

Comment:
This chunk of code should probably belong to  ScopeIdProvider implementation. Wouldn't things look neater if `UpdateFunctionInfoAddress` took a single parameter of type `InstrumentedFunction`?